lib/tasks/limber.rake

```diff
@@ -258,7 +258,7 @@ namespace :limber do
   task create_request_types: [:environment, :create_plates] do
     puts 'Creating request types...'
     ActiveRecord::Base.transaction do
-      %w[WGS LCMB].each do |prefix|
+      %w[WGS LCMB pWGS].each do |prefix|
```

Comment:
Noticed you aren't adding a library type. 
* The Confluence page specified "Library type must be pWGS-384", 
* and I noticed there was an action in the email chain - "ACTION: Lesley will clarify parameters on the requred libarty type"

was this resolved for certain? NPG, the team downstream of us who sort out the sequencing data, tend to care about library types.

Reply:
I will check this

Reply:
Yep you were correct. I have updated to instead use the `pWGS-384` library type